api/services/administration/general/responses/getPolicyAction.yaml

```diff
@@ -0,0 +1,74 @@
+description: Get actions for access policies Response
+content:
+  application/json:
+    schema:
+      allOf:
+        - allOf:
+            - type: object
+              properties:
+                request:
+                  type: object
+                  description: >
+                    Original request properties
+                  properties:
+                    id:
+                      type: string
+                      example: c0ad35b-895f-4f1d-871a-14c2f236cb7a
+                      description: >
+                        Unique ID associated with this request
+                    url:
+                      type: string
+                      example: /setup/access/policyAction?service=ocean&resourcePattern=ocean:ols-*&category=update
+                      description: >
+                        The URL that was used for the request
+                    method:
+                      type: string
+                      example: GET
+                      description: >
+                        The HTTP **METHOD** used for the request
+                    timestamp:
+                      type: string
+                      example: 2022-04-14T05:05:26.044Z
+                      description: >
+                        When the request was received by Spot
+                response:
+                  type: object
+                  description: >
+                    Response package that includes, status, type and any relevant data.
+                  properties:
+                    status:
+                      type: object
+                      properties:
+                        code:
+                          type: integer
+                          example: 200
+                          description: >
+                            HTTP status code assigned to this response
+                        message:
+                          type: string
+                          example: OK
+                          description: >
+                            HTTP status message for the respective status code
+            - type: object
+              properties:
+                response:
+                  type: "object"
+                  properties:
+                    kind:
+                      type: "string"
+                      example: spotinst:setup:policyAction
+                      description: >
+                        The data type of each item in the response
+                    items:
+                      type: "array"
+                      description: >
+                        Array of data objects
+                      items:
```

Comment:
no need to create different example files.
you can add example to the schema file itself. 

you can take an example from `/services/administration/general/schemas/userGroup/getUserGroups.yaml`

Reply:
done